nat/service_iptables.go

```diff
@@ -34,17 +34,9 @@ func (service *serviceIPTables) Add(rule RuleForwarding) {
 }
 
 func (service *serviceIPTables) Start() error {
-	err := service.ipForward.Enable()
-	if err != nil {
```

Comment:
Maybe we should skipp this error here, in high level logic.
But lets have `ipForward.Enable` not swelling errors

Reply:
Reverted error returning